### ISSUE TITLE
selector-list-comma-newline-after fails with LESS mixins when name ends in number

### ISSUE THREAD

**Issue by edg2s** [stylelint/stylelint#1321]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

If your mixin name ends in a number the linter will not detect the commas between arguments as being special.

If the mixin name has no number at the end, the commas between arguments are correctly ignored.

> Which rule, if any, is this issue related to?

selector-list-comma-newline-after

> What CSS is needed to reproduce this issue?

e.g.

```less
 .col3( @a, @b ) {
}
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "selector-list-comma-newline-after": ["always" ]
  }
}
```

> Which version of stylelint are you using?

e.g. `6.3.3`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

LESS

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

Warnings are flagged.

**Comment by jeddy3** [comment]
@edg2s Thanks for the detailed report. I've created [a pending PR](https://github.com/stylelint/stylelint/issues/1361) to ensure all Less mixins are ignored.
